### DISCUSSION TITLE
希望支持payjs(paybob)的充值方式

### DISCUSSION THREAD

**Discussion by sl514** [yangjian102621/chatgpt-plus#88] [规划中的任务]
### ⚠️  确认 issue 是否已存在 ⚠️

- [X] 我已经搜索了现有的问题，没有找到相关 issue。

### 功能描述 📝

payjs(paybob)是国内最早一批的个人业务支持平台，类似现在的虎皮椒。这一个平台用户挺多的，希望作者能支持下。

### 动机 🔦

无法使用payjs支付。

**Comment by yangjian102621** [comment]
收到，感谢反馈。已收入规划中需求。
